lib/demo_web/controllers/user_auth.ex

```diff
@@ -0,0 +1,139 @@
+defmodule DemoWeb.UserAuth do
```

Comment:
would it make sense to move these to controllers/user/auth.ex for all the controllers?

Reply:
That will depend on per application basis, I would prefer to keep things simple than originally start with a complex or nested structure.

Reply:
As someone who is newer to phoenix coming from Rails with devise (thank you for that), what I always appreciated was that devise was highly organized. I had one place to go for many things. Since phoenix encourages contexts (accounts), I think it would be good to encourage auth controllers to be together. I think your goal of adding a default auth generator is fantastic, but if the wrong defaults are there, it will be confusing to many. I couldn't at first tell that the accounts context and the users controller were related. Maybe even demo_web/controllers/accounts to make the learning part easier.